administrator/components/com_templates/tmpl/template/default.php

```diff
@@ -303,6 +303,13 @@
 <?php echo JHtml::_('bootstrap.addTab', 'myTab', 'description', JText::_('COM_TEMPLATES_TAB_DESCRIPTION')); ?>
 <?php echo $this->loadTemplate('description'); ?>
 <?php echo JHtml::_('bootstrap.endTab'); ?>
+
+<?php if ($this->pluginState['plg_installer_override']->enabled) : ?>
```

Comment:
Does the PluginHelper class not provide a method for it?

Reply:
@laoneo As, I can see PluginHelper class not provide a method for it. But, if you know then how?